plugins/builds/update.js

```diff
@@ -85,6 +85,11 @@ module.exports = () => ({
 
                     return eventFactory.get(build.eventId).then(event => ({ build, event }));
                 }).then(({ build, event }) => {
+                    // If payload only has statusMessage
+                    if (!desiredStatus && statusMessage) {
```

Comment:
is this `build.statusMessage`?

Reply:
this statusMessage is from payload: https://github.com/screwdriver-cd/screwdriver/blob/fef6c46497a34b651b2b09868f2d66e32a57e666/plugins/builds/update.js#L30

Reply:
ah, missing the assignment!  fixed